pkg/framework/spi/crypto/api.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
```

Comment:
I'm not seeing any functions that actually supply a Keyset handle. How would I use this API?

Reply:
that would be in the KMS implementation.. for now unit tests of the crypto implementation shows an example of how a key handle can be obtained (from Tink)

Reply:
If you are abstracting tink, you would need interfaces in the SPI.

Reply:
replaced interface arguments of `*keyset.Handle` to `interface{}` in all interfaces functions

Reply:
done